features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos1
+    Then the image is listed as active
+    And this image has non-empty ID
+    And its ID can be used to show the image info
+    When I delete the image
```

Comment:
Should it? I wanted to do some cleanup after the scenario is done, so nothing created for testing is left there. So I actually think it is better to delete it right after the test finishes. @vmoravec what do you think?